integration/class-enable-mastodon-apps.php

```diff
@@ -44,6 +44,7 @@ public static function init() {
 		\add_filter( 'mastodon_api_statuses', array( self::class, 'api_statuses_external' ), 10, 2 );
 		\add_filter( 'mastodon_api_status_context', array( self::class, 'api_get_replies' ), 10, 3 );
 		\add_action( 'mastodon_api_update_credentials', array( self::class, 'api_update_credentials' ), 10, 2 );
+		\add_action( 'mastodon_api_submit_status_text', array( '\\Activitypub\\Mention', 'the_content' ), 10, 2 );
```

Comment:
```suggestion
		\add_action( 'mastodon_api_submit_status_text', array( Mention::class, 'the_content' ), 10, 2 );
```
Let's go with the constant to align with other uses in. the plugin. The Mention class might have to be imported for the suggestion to work

Reply:
Thanks